app/models/concerns/streamable.rb

```diff
@@ -6,36 +6,38 @@ module Streamable
   included do
     has_one :stream_entry, as: :activity
 
-    def title
-      super
+    after_create do
+      account.stream_entries.create!(activity: self, hidden: hidden?) if needs_stream_entry?
     end
+  end
 
-    def content
-      title
-    end
+  def title
```

Comment:
IMO, `#title` , `#target` and `#thread` are not needed because it calls only `super` 🤔